packages/layerzero-v2/evm/oapp/test/TestHelper.sol

```diff
@@ -64,121 +64,127 @@ contract TestHelper is Test, OptionsHelper {
         executorValueCap = _valueCap;
     }
 
+    struct EndpointSetup {
+        EndpointV2[] endpointList;
+        uint32[] eidList;
+        address[] sendLibs;
+        address[] receiveLibs;
+        address[] signers;
+        PriceFeed priceFeed;
+    }
+
+    struct LibrarySetup {
+        SendUln302 sendUln;
+        ReceiveUln302 receiveUln;
+        Executor executor;
+        DVN dvn;
+        ExecutorFeeLib executorLib;
+        DVNFeeLib dvnLib;
+    }
+
+    struct ConfigParams {
+        IExecutor.DstConfigParam[] dstConfigParams;
+        IDVN.DstConfigParam[] dvnConfigParams;
+    }
+
+    EndpointSetup endpointSetup;
+
     /**
      * @dev setup the endpoints
      * @param _endpointNum num of endpoints
      */
     function setUpEndpoints(uint8 _endpointNum, LibraryType _libraryType) public {
```

Comment:
/**
 * @dev Deploys and configures multiple endpoints for cross-chain messaging.
 * Initializes endpoint contracts, registers libraries, and configures price feeds.
 *
 * @param _endpointNum Number of endpoints to deploy.
 * @param _libraryType Type of message library to use (UltraLightNode or SimpleMessageLib).
 */